runtime/parser2/expression.go

```diff
@@ -572,6 +572,13 @@ func parseCreateExpressionRemainder(p *parser, token lexer.Token) *ast.CreateExp
 	}
 }
 
+// Invocation Expression Grammar:
+//
+// invocation
+// : ('<' ( typeAnnotation (',' typeAnnotation )* )? '>')?
+// '(' ( argument (',' argument)* )? ')'
+// ;
+//
```

Comment:
Good idea to add the grammar! Given that this isn't yet able to parse type arguments, maybe leave them out to avoid confusion:

```suggestion
// Invocation Expression Grammar:
//
//     invocation : '(' ( argument ( ',' argument )* )? ')'
//
```